ChiModules/KEigenvalueSolver/initialize_precursors.cc

```diff
@@ -44,12 +44,12 @@ void KEigenvalue::Solver::InitializePrecursors()
           size_t j_map = precursor_map[xs_id][j];
 
           // ----- Initialize precursors
-          double coeff = xs->gamma[j] / xs->lambda[j];
+          double coeff = xs->precursor_gamma[j] / xs->precursor_lambda[j];
```

Comment:
refer to my comment/plea for precursor_gamma to be changed to precursor_yield

Reply:
Done